internal/service/dynamodb/contributor_insights.go

```diff
@@ -0,0 +1,161 @@
+package dynamodb
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
+)
+
+func ResourceContributorInsights() *schema.Resource {
+	return &schema.Resource{
+		CreateWithoutTimeout: resourceContributorInsightsCreate,
+		ReadWithoutTimeout:   resourceContributorInsightsRead,
+		DeleteWithoutTimeout: resourceContributorInsightsDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"index_name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Optional: true,
+			},
+			"table_name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+			"status": {
```

Comment:
Is the `status` useful for the practitioner? We now tend to not include attributes like `status` or `updated_date` in the resource as they often change outside the control of the resource's Terraform configuration. They are often included in any parallel data source.

Reply:
Removed `status`